keys.py

```diff
@@ -8,4 +8,4 @@ class keys(metaclass=ABCMeta):
     columns = "columns"
     pk = "primary_key"
     idx_key = "Index_keys"
-    consistency = "Consistency"
\ No newline at end of file
+    consistency = "Consistency"
```

Comment:
Let’s should be capital letter 
IDX_KEY
PK

Reply:
Ok we will adjust this point but why do they have to be capitalized??

Reply:
Constant in python and Java is better to follow this convention
https://realpython.com/python-constants/#creating-a-dedicated-module-for-constants